legacy_utest/CMakeLists.txt

```diff
@@ -3,10 +3,18 @@ include_directories( ${PROJECT_SOURCE_DIR} )
 
 file(COPY ${CMAKE_SOURCE_DIR}/legacy_utest/txt DESTINATION "${CMAKE_BINARY_DIR}/legacy_utest/")
 
+# Required for new cmake versions
```

Comment:
The problem is that the policy has changed from the version of CMake 3.5 to the latest version, 3.9.

This means that the old policy is currently not supported by the new version and an error happens when creating the google test with cmake.
If I don't set that policy to new the next message is displayed in CMake:
-----------------------------------------------------------------------------------------------------
CMake Warning (dev) at /home/lab1/cmake-3.9.0-rc3-Linux-x86_64/share/cmake-3.9/Modules/GoogleTest.cmake:127 (if):
   Policy CMP0057 is not set: Support new IN_LIST if() operator.  Run "cmake
   --help-policy CMP0057" for policy details.  Use the cmake_policy command to
   set the policy and suppress this warning.

   IN_LIST will be interpreted as an operator when the policy is set to NEW.
   Since the policy is not set the OLD behavior will be used.
 Call Stack (most recent call first):
   unit_tests/CMakeLists.txt:35 (GTEST_ADD_TESTS)
 This warning is for project developers.  Use -Wno-dev to suppress it.

 **CMake Error** at /home/lab1/cmake-3.9.0-rc3-Linux-x86_64/share/cmake-3.9/Modules/GoogleTest.cmake:127 (if):
   if given arguments:

     "utest_grppi" "IN_LIST" "allKeywords"

   Unknown arguments specified
 Call Stack (most recent call first):
   unit_tests/CMakeLists.txt:35 (GTEST_ADD_TESTS)
-----------------------------------------------------------------------------------------------------
The only way I have found so far is setting the policy to NEW.